Luna/Luna/AddPeriodViewController.swift

```diff
@@ -104,6 +92,58 @@ class AddPeriodViewController: UIViewController
         setLabelDates()
     }
     
+    fileprivate func onSavePressed()
+    {
+        let canSave = checkDate()
+        
+        if canSave
+        {
+            createPeriodObject()
+            delegate?.onLastCycleChange()
+            delegate?.onDismissAddPeriod()
+        }
+    }
+    
+    fileprivate func checkDate() -> Bool
+    {
+        
+        let selectDate = addPeriodViewModel.Start
+        let endDate = addPeriodViewModel.End
+     
+        let order = Calendar.current.compare(selectDate, to: endDate, toGranularity: .day)
+        
+        switch order
+        {
+            case .orderedDescending:
+                let alert = UIAlertController( title: Constants.InterfaceBuilderStrings.pastDateTitle,
+                                               message: Constants.InterfaceBuilderStrings.pastDateMessage,
+                                               preferredStyle: .alert )
+                alert.addAction(UIAlertAction(title: "Ok", style: .default, handler: { (action: UIAlertAction!) in
+                    alert.dismiss(animated: true, completion: nil)
+                }))
+                present( alert, animated: true, completion: nil )
+                print("DESCENDING")
+                return false
+            
+            case .orderedAscending:
+                print("ASCENDING")
+                return true
+            
+            case .orderedSame:
+                let alert = UIAlertController( title: Constants.InterfaceBuilderStrings.sameDateTitle,
```

Comment:
Can we build a `class Alert` that has an initializer to customize an alert? check below...